lib/modules/manager/maven/extract.ts

```diff
@@ -498,6 +543,13 @@ export async function extractAllPackageFiles(
         );
         additionalRegistryUrls.push(...registries);
       }
+    } else if (packageFile === '.mvn/extensions.xml') {
```

Comment:
the issue suggested to first parse XML and check xmlns here

Reply:
https://github.com/renovatebot/renovate/issues/28843#issuecomment-2094917565

Reply:
I'm ok with it using the filename, maybe do "endsWith" in case someone has this in a subdirectory